docs/en/02_Logging_In.md

```diff
@@ -1,7 +1,5 @@
-<!--
-pagenumber: 1
-title: Introduction & Logging In
--->
+title: Introduction & logging In
```

Comment:
Just a small thing, I wouldn't call this an Introduction. Just call it Logging In. Also change the Title below : )